frontend/src/components/widgets/FileUploader/FileUploader.tsx

```diff
@@ -405,11 +413,37 @@ class FileUploader extends React.PureComponent<Props, State> {
     )
   }
 
+  /**
+   * If we're part of a clear_on_submit form, this will be called when our
+   * form is submitted. Restore our default value and update the WidgetManager.
+   */
+  private onFormCleared = (): void => {
+    this.setState({ files: [] }, () => {
+      const newWidgetValue = this.createWidgetValue()
+      if (newWidgetValue === undefined) {
```

Comment:
`newWidgetValue == null`? 
Just to be consistent with the rest of the codebase.